common.gypi

```diff
@@ -269,23 +269,11 @@
     'msvs_settings': {
       'VCCLCompilerTool': {
         'StringPooling': 'true', # pool string literals
-        'DebugInformationFormat': 3, # Generate a PDB
+        'DebugInformationFormat': 1, # /Z7 embed info in .obj files
```

Comment:
We currently publish the pdb, e.g. https://nodejs.org/dist/v10.9.0/win-x64/node_pdb.zip so this is semver-major?

cc @nodejs/platform-windows @nodejs/diagnostics

Reply:
See also https://github.com/nodejs/node/blob/4cea01a41027b5d011232da352a9f165221e74a7/vcbuild.bat#L372-L378